server/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js"
+  },
+  "keywords": [],
+  "author": "Sana",
+  "license": "ISC",
+  "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
+    "lodash": "^4.17.21",
```

Comment:
i don't think you are actually using "lodash" dependency.
please avoid binding unneccesary resources, they add extra maintenance woork and increase chances of future deployment failure in case if some of dependecies gets issues.